index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kyle Hillman's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="stylesheet" href="styles/fonts.css" />
+    <link rel="stylesheet" href="styles/reset.css" />
```

Comment:
Your `reset.css` should **always** come first when you include your CSS files.  CSS files load in the order they appear in the HTML file, so right now the reset is loading *last*, and is likely overriding some of the styles in `style.css`.